packages/eslint-plugin-pf-codemods/test/rules/v5/contextSelector-update-deprecatedPath.js

```diff
@@ -0,0 +1,24 @@
+const { createMoveSpecifiersTester } = require("../../testHelpers");
+const importsToMove = [
+  { name: "BadgeToggle", type: "component" },
+  { name: "Dropdown", type: "component" },
+  { name: "DropdownPosition", type: "enum" },
+  { name: "DropdownDirection", type: "enum" },
+  { name: "DropdownContext", type: "component" },
+  { name: "DropdownArrowContext", type: "component" },
+  { name: "DropdownGroup", type: "component" },
+  { name: "DropdownItem", type: "component" },
+  { name: "DropdownMenu", type: "component" },
+  { name: "DropdownSeparator", type: "component" },
+  { name: "DropdownToggle", type: "component" },
+  { name: "DropdownToggleAction", type: "component" },
+  { name: "DropdownToggleCheckbox", type: "component" },
+  { name: "DropdownWithContext", type: "component" },
+  { name: "KebabToggle", type: "component" },
+];
```

Comment:
Just need to copy+paste the `importsToMove` array from the rules/v5 file to here.